### PR TITLE
Remove pachd's init container

### PR DESCRIPTION
**Pull request by acohen4**

### CODE REVIEW

File: etc/helm/pachyderm/templates/pachd/deployment.yaml [hunk 1]
```diff
@@ -236,8 +236,4 @@ spec:
         secret:
           secretName: {{ required "If pachd.tls.enabled, you must set pachd.tls.secretName" .Values.pachd.tls.secretName | quote }}
       {{- end }}
-      initContainers:
-      - name: init-etcd
-        image: busybox:1.28
-        command: ['sh', '-c', "until wget -O- http://etcd:2379/version > /dev/null; do echo waiting for etcd; sleep 2; done"]
```
Comment (PFedak) [inline etc/helm/pachyderm/templates/pachd/deployment.yaml:242]: look like we already have a 3 minute wait in pachd to connect to etcd that most operations block on, any idea if anything else relied on this?

**Review by PFedak — APPROVED** [review]
I don't think I know enough to dispute this, but it makes sense as far as I can tell

File: etc/helm/pachyderm/templates/pachd/deployment.yaml (hunk 1 above)
Comment (acohen4) [reply etc/helm/pachyderm/templates/pachd/deployment.yaml:242]: where do we set the 3 minute wait?

**Comment by acohen4** [comment]
> I don't think I know enough to dispute this, but it makes sense as far as I can tell

looking through it, the etcd client creation should actually block on successful connection. This also happens after we connect to postgres which usually takes longer to setup anyway

File: etc/helm/pachyderm/templates/pachd/deployment.yaml (hunk 1 above)
Comment (PFedak) [reply etc/helm/pachyderm/templates/pachd/deployment.yaml:242]: https://github.com/pachyderm/pachyderm/blob/425ef6c1f55f030fa26d923ef788dcce4f82e80b/src/internal/serviceenv/service_env.go#L246

**Pull request merged**